PurchasesCoreSwift/SubscriberAttributes/AttributionDataMigrator.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  AttributionDataMigrator.swift
+//  PurchasesCoreSwift
+//
+//  Created by César de la Vega on 6/16/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCAttributionDataMigrator) public class AttributionDataMigrator: NSObject {
+
+    @objc public func convertAttributionDataToSubscriberAttributes(
+            attributionData: [String: Any], network: Int
```

Comment:
nitpick: 
I'd either name this 
```swift
convertAttributionDataToSubscriberAttributes(_ attributionData:
```
or 

```swift
convertToSubscriberAttributes(attributionData:
```
so that when called in swift, it's naming won't look redundant: 
```swift
// before
migrator.convertAttributionDataToSubscriberAttributes(attributionData: data, ...

// after, first approach
migrator.convertAttributionDataToSubscriberAttributes(data, ...

// after, second approach
migrator.convertToSubscriberAttributes(attributionData: data, ...

```